src/main/kotlin/no/nav/familie/ef/søknad/mapper/Oversetter.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.familie.ef.søknad.mapper
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.slf4j.LoggerFactory
+
+val kontekst = ThreadLocal<Språk>()
+
+fun String.hentTekst(): String = Oversetter.hentTekst(this)
```

Comment:
Synes nøklene våre virker litt "skjøre" når de er fulle setninger med tegnsetting. 

Èn mulighet er å flytte tekstene fra egen json-fil til enums og dermed alltid ha statisk typesikring på at enum/oversettelsen finnes. 

Vet ikke hva andre tenker rundt dette? Enumen kan jo bli litt "uryddig" når den skal inneholde tre språk, men det er en begrenset mengde tekster vi skal dytte inn her uansett. 

Eksempel;

```

enum class SpråkTekster(val norsk:String, val engelsk:String) {
    Fødselsnummer("Fødselsnummer", "National identity number"),
    DatoMottatt("Dato mottatt", "Date received")
}
private fun SpråkTekster.hentTekst(): String {
    return when(språk()) {
        Språk.EN -> this.engelsk
        Språk.NB -> this.norsk
    }
}
// i bruk
DatoMotatt.hentTekst()
```

Reply:
Ja jeg er også usikker på bruket av disse tekstene som nøkler. Tror tanken var att raskt få ut en fiks på att oversettelse av alle tekster, men det skaper ju egentligen teknisk gjeld. 
Er vel ca 40 tekster som må inn i en enum, er ju mulig det 👍

Reply:
Jeg tester :)

Reply:
Fikset. Det ble bedre :) 
takk for inspill.